test/e2e/systemd_test.go

```diff
@@ -59,7 +59,7 @@ WantedBy=multi-user.target
 			Expect(stop.ExitCode()).To(Equal(0))
 		}()
 
-		create := podmanTest.Podman([]string{"create", "--name", "redis", "redis"})
+		create := podmanTest.Podman([]string{"create", "--name", "redis", redis})
```

Comment:
Just double-checking is removing the quotes here intended/valid?

Reply:
should be yes

Reply:
@TomSweeneyRedHat @baude, late followup, yes, unquoting "redis" is deliberate: that gives us a variable that references an FQIN.